anchor/common/ssv_types/src/committee.rs

```diff
@@ -0,0 +1,23 @@
+use crate::{Operator, OperatorID, OperatorPublicKey};
+use types::Domain;
+
+/// Unique identifier for a committee.
+pub type CommitteeID = u64;
```

Comment:
Same here as operator id. (See below)

Its a bit cumbersome, but it does pay dividends in the future